eZ/Bundle/EzPublishCoreBundle/ApiLoader/RepositoryConfigurationProvider.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * This file is part of the eZ Publish Kernel package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace eZ\Bundle\EzPublishCoreBundle\ApiLoader;
+
+/**
+ * The repository configuration provider.
+ */
+class RepositoryConfigurationProvider extends StorageRepositoryProvider
+{
+}
```

Comment:
If the class doesn't do anything, then we most likely don't need it.
There is no BC need for `StorageRepositoryProvider` AFAICT, so you can rename it.